### PR TITLE
Feature/deployment into pod#9

### PR DESCRIPTION
**Pull request by oliver-tarrant-tessella**
* Deployment scripts to simplify the task of deploying Piezo in a pod. 
* Add configuration file to allow local deployment on your system
* Wiki has also been updated with instructions to deploy Piezo both locally and in a pod and how to set up your local development environment

## Stories covered
#9

### CODE REVIEW

File: code/package_piezo.sh [hunk 1]
```diff
@@ -0,0 +1,3 @@
+sudo docker build -t web_app:latest .
+sudo docker tag web_app:latest host-172-16-113-64.nubes.stfc.ac.uk/piezo-resources/web_app:latest
+sudo docker push host-172-16-113-64.nubes.stfc.ac.uk/piezo-resources/web_app:latest
```
Comment (robert-clegg-tessella) [inline code/package_piezo.sh]: There's no guarantee that the target IP address will stay as `host-172-16-113-64.nubes.stfc.ac.uk` forever - can we at least pull this out to one variable that gets reused?

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+COPY piezo_web_app/requirements.txt ./
+RUN pip install --no-cache-dir -r requirements.txt
+
+WORKDIR /piezo_web_app/
+
+ENV PATH /piezo_web_app:$PATH
+ENV PYTHONPATH /piezo_web_app/
+
+EXPOSE 8888
```
Comment (robert-clegg-tessella) [inline Dockerfile]: Please add a comment that this is the port number (if indeed that's what it is!)

File: deploy_piezo.yaml [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    matchLabels:
+      run: piezo
+  template: 
+    metadata:
+      labels:
+        run: piezo
+    spec:
+      containers:
+      - name: piezo
+        image: host-172-16-113-64.nubes.stfc.ac.uk/piezo-resources/web_app:latest
```
Comment (robert-clegg-tessella) [inline deploy_piezo.yaml]: This (and other) references to `host-172-16-113-64.nubes.stfc.ac.uk` needs to be pulled out into a `var` or similar

File: package_piezo.sh [hunk 4]
```diff
@@ -0,0 +1,3 @@
+sudo docker build -t web_app:latest .
```
Comment (robert-clegg-tessella) [inline package_piezo.sh]: Is this file a duplicate of `code/package_piezo.sh`?

**Comment by oliver-tarrant-tessella** [comment]
Pull request also removes driver core limit argument as this proves inconsistent and extremely temperamental for development. Affect is minimal on cpu usage but speeds up spark driver pods. Will look to reintroducing later if appropriate

**Pull request merged**